source/uri-options/tests/srv-service-name-option.json

```diff
@@ -0,0 +1,24 @@
+{
+  "tests": [
+    {
+      "description": "SRV URI with custom srvServiceName",
+      "uri": "mongodb+srv://test3.test.build.10gen.cc/?srvServiceName=customname",
```

Comment:
I'll have to speak with the BUILD team about setting up a custom SRV record with `customname` as a service name for this spec test to work. Will update with a ticket number on this thread.

Reply:
I assume we need a custom SRV record because testing with `srvServiceName=mongodb` wouldn't really distinguish between a driver ignoring `srvServiceName` altogether and using the value explicitly. Is that correct?

Reply:
Will tests also be added to the [SRV spec](https://github.com/mongodb/specifications/tree/master/source/initial-dns-seedlist-discovery/tests)?